src/features/query-bar/__tests__/Condition-test.js

```diff
@@ -76,7 +78,7 @@ describe('features/query-bar/components/filter/Condition', () => {
     });
 
     describe('handleValueChange()', () => {
-        const condition = initialCondition;
+        const condition = invalidCondition;
```

Comment:
The new name `invalidCondition` is more descriptive and helpful.

I assume that all tests are compatible with the concept of "invalid condition".

Reply:
Yes they are compatible